x-pack/plugins/security_solution/common/ecs/index.ts

```diff
@@ -70,4 +70,5 @@ export interface Ecs {
   Memory_protection?: MemoryProtection;
   Target?: Target;
   dll?: DllEcs;
+  'kibana.alert.workflow_status'?: 'open' | 'acknowledged' | 'in-progress' | 'closed';
```

Comment:
this all needs to be adapted to the new schema... pull in https://github.com/elastic/kibana/pull/107713 ?